staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go

```diff
@@ -45,16 +45,19 @@ type ServerRunOptions struct {
 	MinRequestTimeout           int
 	TargetRAMMB                 int
 	WatchCacheSizes             []string
+
+	admissionPlugins *admission.Plugins
```

Comment:
Comment that we want to push this (or something similar) down into recommended options.